sw/device/lib/base/mmio.h

```diff
@@ -234,6 +234,46 @@ inline void mmio_region_nonatomic_set_mask32(mmio_region_t base,
   mmio_region_write32(base, offset, value);
 }
 
+/**
+ * Masked Register field for use in 32-bit bitfields.
+ *
+ * |index| represents a shift in bits starting form the 0 bit of a given 32-bit
+ * bitfield. It is used to zero the memory inside this bitfield by applying an
+ * inverted |mask| at the |index| offset. |value| is then capped by applying
+ * |mask| and is shifted to the |index| bits.
```

Comment:
Please use backticks or `@p` rather than pipes.

Reply:
Okay, I don't know what the final ruling on this is, so I will keep using @p for functions and use backticks for enums and structs.

Reply:
Done.